libs/designsystem/src/lib/components/dropdown/dropdown.component.scss

```diff
@@ -12,6 +12,11 @@ $min-screen-width: 375px;
     border-radius: utils.$border-radius-round;
   }
 
+  // // TODO: Improve this - an attempt to remove focus outline on iOS
+  // &:focus {
+  //   outline: none;
+  // }
+
```

Comment:
Don't forget to remove this block comment 😎

Reply:
Ahh nice catch!